docs/application/native/guides/ui/tizen-ws-shell/index.md

```diff
@@ -0,0 +1,76 @@
+# Tizen Window System Shell
+
+The Tizen Window System Shell (TZSH) is an interface for manipulating windows of the system Graphical User Interface (GUI) services such as the Quickpanel and the Softkey. It provides C libraries that allow you to control windows of system GUI services and get notifications about the state changes of each window within an application.
+
+The system GUI service runs different processes. Therefore, all the operations of TZSH are performed using the inter-process communication (IPC). It is needed to enable communication between system GUI service processes and separated user application process using the display server. The following figure illustrates a simple software architecture of the TZSH:
+
+**Figure: Layer diagram for the Tizen window system shell**
+
+![Layer diagram for the Tizen window system shell](./media/tzsh_arch.png)
+
+To support each system GUI service in Tizen, TZSH provides the following libraries:
+
+- **[TZSH-Quickpanel](./tzsh-quickpanel.md)**: For the Quickpanel window that shows notifications and system setup widgets.
+
+- **[TZSH-Softkey](./tzsh-softkey.md)**: For the Softkey window that shows the device back and home buttons.
+
+All applications do not need to use the TZSH's functionalities. However, in some cases, certain applications may require to perform manipulation of system GUI service window. For example, media player application needs to close the Quickpanel window during playback of video. In this case, you can use the TZSH-Quickpanel library.
```

Comment:
If we read “Not all applications require to use the TZSH's functionalities.” and  “All applications do not need to use the TZSH's functionalities.” separately, then I totally agree with you that the meaning has changed.

But, the complete suggestion given was: 
All applications do not need to use the functionalities of TZSH. However, in some cases, certain applications may require to perform manipulation of system GUI services window.

The second part, from "However...", gives the same meaning as suggested by you.

I think @sabeena21  rephrased this to avoid, starting a sentence with a negative tone. 
However, if you still feel the original one is better. Then we would like to go as per your suggestion.

Reply:
I agree that first "All" can cause useless confusion while reading. And also, I think using just "Application" can confusing too. So, Suggestion 2 is looks good to me. I modified as your suggestion 2.

Reply:
@mijin85cho 
I agree with you. 
I would like to go with Suggestion 2.
@juns-kim 
Thanks for updating the second suggestion as per @mijin85cho.